Makefile

```diff
@@ -50,70 +49,61 @@ autoloads:
 
 # Create .tgz and .zip files only
 # GNUTAR=gtar make tarballs
-tarballs: $(ESSDIR)
+.PHONY: rel-tarballs
+rel-tarballs: rel-staging
```

Comment:
Why the rename?

Reply:
`rel` is the umbrella make directive by which mmaechler creates the self-contained tarball.  I think it's important to associate the `tarballs` target with `rel`.